<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPlayerWidget</class>
 <widget class="QWidget" name="MainPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#toolbar {
	border-top: 1px solid rgba(255, 255, 255, 20);
	border-bottom: 1px solid rgba(0, 0, 0, 50);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="toolbar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="Player::PlayerLabel" name="labelA" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Player::PlayerLabel" name="labelB" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="Player::AudioPlayerWidget" name="playerA" native="true"/>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(45, 45, 45);

</string>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Player::AudioPlayerWidget" name="playerB" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Player::AudioPlayerWidget</class>
   <extends>QWidget</extends>
   <header>audioplayerwidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Player::PlayerLabel</class>
   <extends>QWidget</extends>
   <header location="global">playerlabel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
